src/marshaller/HipieDDL.js

```diff
@@ -782,8 +782,10 @@
     Visualization.prototype.setWidget = function (widget) {
         this.widget = widget;
         this.events.setWidget(widget);
-        var columns = this.source.getColumns();
-        this.widget.columns(columns);
+        if (this.widget.columns) {
```

Comment:
I am not sure this check is even needed since all our widgets have a columns method - I think I might simply remove it?